client/components/mma/cancel/cancellationSaves/SelectReason.tsx

```diff
@@ -196,11 +214,15 @@ export const SelectReason = () => {
 
 		try {
 			setIsSubmitting(true);
-			const response = await cancellationCaseFetch(
-				selectedReasonId,
-				productType,
-				productDetail,
-			);
+			const response = await Promise.all([
```

Comment:
I'm still not 100% sure what happens when one of these calls fails - or what should happen

Reply:
I'll test in code

Reply:
if the calls fail the user will still navigate to the next page

Reply:
Yep, just tested it. That's better UX than showing an error there.